pkg/server/server.go

```diff
@@ -1564,3 +1564,19 @@ func (s *Server) checkCreateStoreAuthz(ctx context.Context) error {
 	}
 	return nil
 }
+
+// checkWriteAuthz checks the authorization for modules if they exist, otherwise the store on write requests.
+func (s *Server) checkWriteAuthz(ctx context.Context, req *openfgav1.WriteRequest, typesys *typesystem.TypeSystem) error {
+	if s.authorizer != nil {
```

Comment:
Not for this PR: Could we make `s.authorizer` an interface so we don't have to make this nil check here? The default implementation could be a NoOpAuthorizer that allows all requests. And then if you `enable-access-controls` you instantiate your custom Authorizer